deploy/test/test_cases/TEST_ID_19_helm_multiple_provider_same_secret.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This test verifies that two Secrets Provider Jobs deploy successfully in the same namespace, can access the same Conjur secret, and update the same K8s Secret
+# TODO verify Job start up and completed logs
+
+setup_helm_environment
+
+pushd ../../../
+  fill_helm_chart
+  helm install -f "secrets-provider/ci/test-values.yaml" \
+    secrets-provider ./secrets-provider \
+    --set-file environment.conjur.sslCertificate.value="deploy/test/test_cases/conjur.pem"
+
+  export SECRETS_PROVIDER_ROLE=another-secrets-provider-role
+  export SECRETS_PROVIDER_ROLE_BINDING=another-secrets-provider-role-binding
+  export SERVICE_ACCOUNT=another-secrets-provider-service-account
+  export SECRETS_PROVIDER_SSL_CONFIG_MAP=another-secrets-provider-ssl-config-map
+  fill_helm_chart "another-"
+  helm install -f "secrets-provider/ci/another-test-values.yaml" \
+    another-secrets-provider ./secrets-provider \
+    --set-file environment.conjur.sslCertificate.value="deploy/test/test_cases/conjur.pem"
+popd
+
+# Deploy app to test against
+deploy_helm_app
+
+# Check for Job completion
+helm_chart_name="secrets-provider"
+$cli_with_timeout wait --for=condition=complete job/$helm_chart_name
+
+helm_chart_name="another-secrets-provider"
+$cli_with_timeout wait --for=condition=complete job/$helm_chart_name
+
+pod_name=$($cli_with_timeout get pods --namespace=$APP_NAMESPACE_NAME --selector app=test-env --no-headers | awk '{print $1}')
+verify_secret_value_in_pod $pod_name "TEST_SECRET" "supersecret"
```

Comment:
Lets verify that the second secret-provider have the same secret as well

Reply:
Only the app gets the secret in it's environment, not the secrets provider

Reply:
Secrets Providers just update the secret with conjur secrets

Reply:
I meant to verify with different app (app=another-test-env) that we have the same secret

Reply:
we don't have a second app deployed. Just a second secrets provider so it looks like
2 Secrets Provider 1 App because we are testing that 2 Secrets Providers can exist in same namespace and update 1 secret

Reply:
Are you suggesting we deploy another app? If so, it looks similar to test 18 no?

Reply:
Sorry my mistake.. :)